python/mxnet/visualization.py

```diff
@@ -166,31 +166,44 @@ def print_layer_summary(node, out_shape):
 
 def plot_network(symbol, title="plot", save_format='pdf', shape=None, node_attrs={},
                  hide_weights=True):
-    """Convert a symbol to a dot object for visualization.
+    """Creates a visualization (Graphviz digraph object) of the given computation graph.
+    Graphviz must be installed for this function to work.
 
     Parameters
     ----------
-    title: str
-        Title of the dot graph.
+    title: str, optional
+        Title of the generated visualization.
     symbol: Symbol
-        symbol to be visualized
-    shape: dict
-        If supplied, the visualization will include the shape
-        of each tensor on the edges between nodes.
-        This is a dict of shapes, str->shape (tuple), given input shapes
-    node_attrs: dict
-        dict of node's attributes.
-        for example:
-            node_attrs={"shape":"oval","fixedsize":"fasle"}
-            means to plot the network in "oval"
-    hide_weights: bool
-        If True (default), then inputs with names like `*_weight`
-        or `*_bias` will be hidden.
+        A symbol from the computation graph. The generated digraph will visualize the part
+        of the computation graph required to compute `symbol`.
+    shape: dict, optional
+        Specifies the shape of the input tensors. If specified, the visualization will include
```

Comment:
If you have specific suggestions on how to rewrite any of them, please let me know.